lsm-tree.cabal

```diff
@@ -452,7 +505,7 @@ benchmark rocksdb-bench-wp8
   hs-source-dirs: bench/macro src-extras
   main-is:        rocksdb-bench-wp8.hs
 
-  if !(os(linux) && flag(rocksdb))
+  if (!(os(linux) && flag(rocksdb)) || !impl(ghc >=9.2.0))
```

Comment:
`if !os(linux) || !flag(rocksdb) || !impl(ghc >=9.2)` is easier to understand; or
`if !(os(linux) && flag(rocksdb) && impl(ghc >=9.2)`.